### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by Joel-100**
- Add mobile menu
- Style mobile menu
- Add Javascript code
- Change linters

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-unused-vars
+function clickToSwitch() {
+  const switchD = document.getElementById('switch-display');
```
Comment (tufoinnkuo10) [inline script.js:3]: - [x] great job so far. Everything looks fine. Kindly reposition your close icon like that of Figma
- [x] please add the size of the close icon, so it is more visible
- [x] please add the font size and weight(600) of the menu links.

yours
Screenshot (423)_LI
figma
Screenshot (421)

File: index.html [hunk 2]
```diff
@@ -20,9 +21,11 @@
           <div class="tools">
             <nav class="main-nav">
               <div class="items">
-                <a class="logo" href="#">My Logo</a>
+                <a class="logo" href="#head">My Logo</a>
                 <div class="mobile" id="harmburger">
```
Comment (tufoinnkuo10) [inline index.html:25]: - [optional] please add the size of the hamburger icon to make it more visible

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @Joel-100 

Great job so far. You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review


tufoin



Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: script.js (hunk 1 above)
Comment (Joel-100) [reply script.js:3]: Completed

**Review by omarsalem7 — APPROVED** [review]
Hi @Joel-100 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**